docker/structures_service.go

```diff
@@ -167,6 +167,12 @@ func flattenContainerSpec(in *swarm.ContainerSpec) []interface{} {
 	if len(in.Isolation) > 0 {
 		m["isolation"] = string(in.Isolation)
 	}
+	if in.TTY {
+		m["tty"] = in.TTY
+	}
+	if in.OpenStdin {
+		m["stdin_open"] = in.OpenStdin
+	}
```

Comment:
This function isn't used at `docker_container` resource, so I think we don't have to update this function in this pull request.

```suggestion
```

Reply:
Thank you. removed